README.md

```diff
@@ -1,14 +1,25 @@
-# latexify_py
-Generates LaTeX math description from Python functions.
+<h1 align = "center">latexify_py</h1>
```

Comment:
- Remove spaces around `=`.
- Style attributes such as `align` is deprecated in HTML and is recommended to be avoided.